src/scss/components/links/_links.scss

```diff
@@ -0,0 +1,59 @@
+.su-link,
+.su-link--action,
+.su-link--external {
+  color: $su-color-digital-red;
+  font-weight: 600;
+
+  &:hover,
+  &:focus,
+  &:active {
+    color: $su-color-black;
+  }
+
+  &::after {
+    background-color: $su-color-digital-red;
+  }
+
+  &.mailto {
+    svg {
+      fill: $su-color-digital-red;
+    }
+  }
+}
+
+.su-button,
+.su-button--big {
+  color: $su-color-white;
+
+  &:hover,
+  &:focus,
+  &:active {
+    color: $su-color-white;
+  }
+}
+
+.su-button--secondary {
+  &:hover,
+  &:focus,
+  &:active {
+    text-decoration: underline;
+  }
+}
+
+.su-local-footer__action-links {
+  a {
+    &::after {
```

Comment:
Hi @jenbreese 

This only targets the arrows after the links and doesn't apply to the link themselves. Is that a requirement here? None of the links in the footer are red.

<img width="254" height="122" alt="Screenshot 2025-08-12 at 6 04 34 PM" src="https://github.com/user-attachments/assets/5e13b2ed-9432-48ea-bc75-4e3619307963" />